src/store/channels-list/saga.ts

```diff
@@ -409,12 +413,17 @@ export function* otherUserJoinedChannel(roomId: string, user: User) {
     return;
   }
 
-  // TODO: Fetch user from zOS if we don't know about them yet
+  if (user.userId === user.matrixId) {
+    user = yield select(userByMatrixId, user.matrixId);
```

Comment:
Yes, it works. Tested with users on separate chrome profiles. 

Actually i don't think "this" code works, it will return just the matrix user. But later we're just loading the conversations (for the other user of create conversation), probably because of this event:

https://github.com/zer0-os/zOS/blob/d61983e31b96eb240bada93dddac6f29d63138d9/src/store/channels-list/saga.ts#L327

If we remove this event, then we'll need to fetch the user from the API. But right now it just works.